flink-python/dev/build-wheels.sh

```diff
@@ -13,6 +13,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# The file is directly downloaded and used by the different connector implementations
```

Comment:
I think it would be better to make a deep copy in `flink-connector-shared-utils` repo because I can't really imagine that Flink and all the connectors require the same behaviour.

Reply:
I think the following functionality is not covered but honestly no idea what is the general consequence:
```
export CFLAGS="-I. -include dev/glibc_version_fix.h"
```